HiFi voice cloning docs (#34)
cartesia-ai/docs: fern/build-with-sonic/capabilities/clone-voices.mdx

2. **Speak as clearly as possible and avoid background noise.** For example, when recording yourself, try to use a high-quality microphone, be in a quiet space, and so on.
3. **Limit your recording to 5 to 10 seconds.** This is the sweet spot—a longer clip will not result in a better clone.
4. **Set `enhance` to `true` when cloning.** This optimizes sample clip quality prior to voice cloning-—improving clone fidelity, especially for lower-quality samples. Note that this may increase overall volume and remove background noises.
5. **Avoid long pauses in the clip.** Too many long pauses could result in the cloned voice drifting from the source clip.
6. **Speak in the target language.** For instance, if you want the cloned voice to speak Spanish, speak Spanish in the recording. If this is not possible, you can use Cartesia's localization feature—available in the playground and in the API—to convert your clone to a different language.

## Hi-Fi Voice Cloning

<Frame background="subtle">
  <img src="/assets/images/hifi-clone.png" />
</Frame>

> Hi-Fi voice cloning is a flavor of instant cloning for creating higher-fidelity clones. This means higher speaker similarity, but noise and other artifacts from the source clip may also get picked up.

Reviewer: ```suggestion
Hi-Fi voice cloning is a type of instant cloning that creates higher fidelity clones, but it's also more sensitive to picking up artifacts and aberrations in the source clip.
```
